docs/source/apidoc.md

```diff
@@ -149,7 +147,13 @@ See Ref. :cite:`Czarnik_2021_Quantum` for more details on these methods.
    :members:
 ```
 
-### Utility Functions
+### Classical Post-Processing
+```{eval-rst}
+.. automodule:: mitiq.shadows.classical_postprocessing
+   :members:
+```
+
+### Classical Shadow Utility Functions
```

Comment:
```suggestion
### Utility Functions
```
Since this is under the "Classical Shadows" level-2 header, I don't think we need to repeat that here.